frontend/server/src/Controllers/Course.php

```diff
@@ -2051,7 +2050,7 @@ public static function apiListSolvedProblems(\OmegaUp\Request $r) : array {
      * Get Problems unsolved by users of a course
      *
      * @param \OmegaUp\Request $r
-     * @return array{status: string, user_problems: array<array-key, array{alias: string, title: string, username: string}>[]}
+     * @return array{status: string, user_problems: array<string, array<int, array{alias: string, title: string, username: string}>>}
```

Comment:
```suggestion
     * @return array{status: string, user_problems: array<string, array{alias: string, title: string, username: string}[]>}
```

hay que preferir `x[]` sobre `array<int, x>` si es un arreglo normal: lo segundo es más bien para indicar un arreglo asociativo donde las llaves son enteros arbitrarios no-necesariamente consecutivos.

Reply:
Listo